Automattic/wp-calypso
# Update the plan features graphics

Closes https://github.com/Automattic/dotcom-forge/issues/9543

## Proposed Changes

This PR updates the plan features graphics to the new WordPress blue.

**Before**

<img width="1056" alt="Screenshot 2024-10-28 at 4 39 50 PM" src="https://github.com/user-attachments/assets/716595c2-e62b-4e73-84fa-ae1c914f237b">

**After**
<img width="1031" alt="Screenshot 2024-10-28 at 4 40 12 PM" src="https://github.com/user-attachments/assets/41146f22-3c9b-412c-a1e4-84f570e5dc66">

## Why are these changes being made?


*

## Testing Instructions



* Navigate to the Calypso Live Link below
* Ensure that you have a WPCOM site with Business plan
* Navigate to `plans/my-plan/{yourBusinessPlanSite`
* Observe that the graphics are now with the new WordPress blue

Figma: NNfIg8OP1Xx0kumwkd2if8-fi-2_15191

## Pre-merge Checklist
